src/themes/index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta name="viewport" content="width=device-width">
+    <link rel="stylesheet" href="../style.css">
+    <link rel="icon" type="image/png" href="../favicon.png" />
+    <meta name="description" content="Search for Srcery implementations, themes and configurations">
```

Comment:
```suggestion
    <meta name="description" content="Search for Srcery implementations, themes, and configurations">
```

I think you need a `,` when you're listing stuff.  I think that is. 🤣